quaint/src/connector/mssql/native/mod.rs

```diff
@@ -60,6 +61,7 @@ pub struct Mssql {
     url: MssqlUrl,
     socket_timeout: Option<Duration>,
     is_healthy: AtomicBool,
+    system_name: &'static str,
```

Comment:
Why store it in the struct if it's a constant that can never change?

```suggestion
```

Reply:
It was to keep consistency with the `postgres/native/mod.rs` connector.

Reply:
I don't think it necessarily has to be consistent with it, it's the postgres connector that is being a weird one and a special case here.